index.html

```diff
@@ -515,13 +531,13 @@
       </p>
       <dl title="interface PushManager" class="idl">
         <dt>
-          Promise&lt;PushSubscription&gt; subscribe ()
+          Promise&lt;PushSubscription&gt; subscribe (optional PushSubscriptionOptions options)
         </dt>
         <dt>
           Promise&lt;PushSubscription?&gt; getSubscription ()
         </dt>
         <dt>
-          Promise&lt;PushPermissionStatus&gt; hasPermission ()
+          Promise&lt;PushPermissionStatus&gt; hasPermission (optional PushSubscriptionOptions options)
```

Comment:
It's perfectly fine to subscribe to push without any options (to get what we call "silent push").